rocAL/rocAL_pybind/examples/rocAL_api_caffe2_reader.py

```diff
@@ -27,77 +27,87 @@
+        if bboxes is not None:
+            bboxes = bboxes.detach().numpy()
+            for (l, t, r, b) in bboxes:
+                loc_ = [l, t, r, b]
+                color = (255, 0, 0)
+                thickness = 2
+                image = cv2.rectangle(image, (int(loc_[0]), int(loc_[1])), (int(
+                    (loc_[2])), int((loc_[3]))), color, thickness)
+        cv2.imwrite("OUTPUT_IMAGES_PYTHON/CAFFE2_READER/DETECTION/" +
+                    str(idx)+"_"+"train"+".png", image)
 
 
 def main():
     args = parse_args()
     # Args
     image_path = args.image_dataset_path
-    _rocal_cpu = False if args.rocal_gpu else True
+    rocal_cpu = False if args.rocal_gpu else True
     batch_size = args.batch_size
-    _rocal_bbox = False if args.classification else True
+    rocal_bbox = False if args.classification else True
     num_threads = args.num_threads
-    local_rank =  args.local_rank
+    local_rank = args.local_rank
     random_seed = args.seed
     display = True if args.display else False
     device = "gpu" if args.rocal_gpu else "cpu"
+    tensor_layout = types.NHWC if args.NHWC else types.NCHW
     num_classes = len(next(os.walk(image_path))[1])
     print("num_classes:: ", num_classes)
     try:
         if args.classification:
-            path= "OUTPUT_IMAGES_PYTHON/NEW_API/CAFFE2_READER/CLASSIFICATION/"
+            path = "OUTPUT_IMAGES_PYTHON/CAFFE2_READER/CLASSIFICATION/"
         else:
-            path= "OUTPUT_IMAGES_PYTHON/NEW_API/CAFFE2_READER/DETECTION/"
+            path = "OUTPUT_IMAGES_PYTHON/CAFFE2_READER/DETECTION/"
         isExist = os.path.exists(path)
         if not isExist:
             os.makedirs(path)
     except OSError as error:
         print(error)
     pipe = Pipeline(batch_size=batch_size, num_threads=num_threads, device_id=local_rank,
-                    seed=random_seed, rocal_cpu=_rocal_cpu)
+                    seed=random_seed, rocal_cpu=rocal_cpu)
     with pipe:
-        if _rocal_bbox:
-            jpegs, labels, bboxes = fn.readers.caffe2(
-                path=image_path, bbox=_rocal_bbox, random_shuffle=True)
+        if rocal_bbox:
+            jpegs, labels, bboxes = fn.readers.caffe2(path=image_path, bbox=rocal_bbox)
         else:
-            jpegs, labels = fn.readers.caffe2(
-                path=image_path, bbox=_rocal_bbox, random_shuffle=True)
+            jpegs, labels = fn.readers.caffe2(path=image_path, bbox=rocal_bbox)
         images = fn.decoders.image(jpegs, output_type=types.RGB, path=image_path, random_shuffle=True)
-        images = fn.resize(images, resize_x=224, resize_y=224)
+        images = fn.resize(images, resize_width=224, resize_height=224, output_layout=tensor_layout)
         pipe.set_outputs(images)
     pipe.build()
-    data_loader = ROCALClassificationIterator(pipe , display=display, device=device)
+    data_loader = ROCALClassificationIterator(pipe, display=0, device=device)
```

Comment:
The display is hardcoded to 0 because there's another draw_patches function in the Iterator which also dumps images. Since we are dumping images in this example, we don't use that function so we set display to 0